rules-topics/08_magic-skills.md

```diff
@@ -544,15 +558,6 @@ In order to cast a spell, you must possess the appropriate Mastery skill, and pu
 | **Incant**     | "I grant you the gift of Life." |
 | **Description** |  This spell will restore a living or dead target to full maximum health, remove all magical, toxin, and physical effects, including beneficial ones, and grants the target temporary immunity to spells, toxins, and physical attacks for 5 seconds. This spell cannot be extended by Limited time Stop.  This is a LIFE effect. |
 
-#### **Resurrection**
-| **Spell**            | **Resurrection** |
-|:---|:---|
-| **Duration**     | One Minute |
-| **Incant**     | "By the spirits, I grant you a resurrection." |
-| **Description** |  This spell takes 1 minutes to cast and requires an untethered spirit. At the beginning of the spell, the target must clearly acknowledge the acceptance of the spell, if they do not, the spell fails to no effect. Once the spell begins, the target's body begins to immediately reform, incurring strain as per a normal resurrection. The caster must be in concentration and unimpeded for the entire casting time of this spell, else the spell will fail. If the caster’s concentration is interrupted, or the reforming body is affected by any detrimental effect, the spell fails and the target immediately suffers a second death. Prior to the casting time the player of the caster, and the player of the untethered spirit, are to verify that the spirit does not have too much strain to resurrect. A marshal should also be present to oversee the process. Once the Resurrection spell is complete, the spirit instantly reforms its body next to the spell caster. The body appears with a full health pool, and all skills renewed as if they had completed a long rest.  This is a LIFE effect. |
-
-
-
 
 ## Nature
```

Comment:
Glowing Light, Barkskin, Stoneskin, & Boon of the Rabbit still appear under Nature.